srbhr/Resume-Matcher
# added dropdown menu to select resume instead of slider

## Pull Request Title
Added Dropdown Menu for Resume and Job Description Selection

## Description
This pull request introduces a dropdown menu for selecting resumes and job descriptions, replacing the previous slider-based selection method. This change enhances user experience and provides a more user-friendly way to choose both resumes and job descriptions.

## Type



## Proposed Changes


- Replaced the slider with a dropdown menu for selecting resumes.
- Implemented a dropdown menu for selecting job descriptions.

## Screenshots / Code Snippets (if applicable)

![image](https://github.com/srbhr/Resume-Matcher/assets/110348311/88db72b3-d243-4d32-9f50-424633ba9c24)
![image](https://github.com/srbhr/Resume-Matcher/assets/110348311/0dc62659-0b5c-4b89-9969-9af2aa64debb)


## How to Test


1. Open the application.
2. Navigate to the resume selection section.
3. Verify that a dropdown menu is available for selecting resumes, where number of resumes are greater than 1.
4. Repeat the same steps for job description selection.

## Checklist
